README.windows.md

```diff
@@ -436,3 +436,17 @@ running the resulting installer.
    inspect the list of loaded DLLs for bash, mintty, or another persistent
    process used to drive the build. Essentially *any* DLL outside of the Windows
    System directory is potential BLODA.
+
+   ### Compiling for Bash on windows
+   Bash on windows is an unsupported platform. However, compilation of julia is
+   possible and functional. The packages required differ from standard ubuntu
```

Comment:
Might be a filesystem bug – these could get triggered by having incorrect file timestamps.